src/logic/operate.js

```diff
@@ -0,0 +1,24 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  const bigNumberOne = new Big(numberOne);
+  const bigNumberTwo = new Big(numberTwo);
+
+  if (operation === '+') {
+    result = bigNumberOne.plus(bigNumberTwo);
+  } else if (operation === '-') {
+    result = bigNumberOne.minus(bigNumberTwo);
+  } else if (operation === 'x') {
+    result = bigNumberOne.times(bigNumberTwo);
+  } else if (operation === '÷') {
+    if (numberTwo === 0) return NaN;
+    result = bigNumberOne.div(bigNumberTwo);
+  } else {
```

Comment:
### [Optional]
It is a good practice to use `switch case` once our condition is more than 3;
kindly check out this article 👉  [JS switch vs if-else](https://medium.com/the-daily-standup/javascript-switch-statement-vs-if-else-if-ee3111793fa0
)